stories/navbar.stories.js

```diff
@@ -0,0 +1,180 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { argument } from './knob-categories';
+
+export default {
+  title: 'DenaliNavbar',
+  component: 'DenaliNavbar',
+  decorators: [withKnobs],
+};
+
+export const Default = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
```

Comment:
Looks like the indent is a little off here. Also, can we make the example responsive by default?